src/metabase/driver/util.clj

```diff
@@ -356,6 +357,9 @@
                 (assoc :visible-if v-ifs*))))
          final-props)))
 
+(def ^:private ^Base64$Decoder base64-decoder
+  (Base64/getDecoder))
```

Comment:
Similar comment here as above, about using the `metabase.util` decoder if possible.